README.md

```diff
@@ -94,7 +99,7 @@ There are now various different ways to let the Python environment know of these
 
 #### `setup.py`
 
-In the package's `setup.py` file, one can specify entry points.  
+In the package's `setup.py` file, one can specify entry points.
```

Comment:
```suggestion
In the package's `setup.py` file, one can specify entry points.  
```
Again, this white space should **not** be removed. It's markdown-ish for (new line, but don't create a new section).